src/System.Security.Cryptography.Encoding/ref/System.Security.Cryptography.Encoding.cs

```diff
@@ -66,4 +66,29 @@ public enum OidGroup
         SignatureAlgorithm = 4,
         Template = 9,
     }
+
```

Comment:
Since it's not yet stable and it doesn't change the netfx compatibility version I don't see a reason to bumpver.